cryptomator/index.md

```diff
@@ -141,6 +141,10 @@ Passwords are saved in the *Credential Manager*. You can view and delete your sa
 
 File transfers require you to unlock the vault again unless you have chosen to save your vault passphrase in the keychain.
 
+### Renaming
+
+Make sure that the vault is locked before it gets renamed. Otherwise the vault will be changed to a normal folder. Using [Smart Synchronization](../mountainduck/connect/sync.md) mode an additional vault with the original name will appear.
```

Comment:
I would add to bullet items for what is expected when renaming a locked|unlocked vault.